src/jsx/Validator.js

```diff
@@ -114,11 +114,11 @@ const Validator = (props) => {
       <div className={'info'}>
         <div className={'small-pad'}>
           <b style={{cursor: 'default'}}>
-            Confirm BIDS data structure:&nbsp;
+            Run BIDS Validator:&nbsp;
           </b>
           <input onClick={validateBIDS}
             type={'button'}
-            value={'Validate BIDS'}/>
+            value={' Validate BIDS '}/>
```

Comment:
Hi @christinerogers, the spaces need to be removed.

Reply:
done